### ISSUE TITLE
Fraud details missing user agent and accept language

### ISSUE THREAD

**Issue by rramsden** [komoju/spree_komoju#56]
It seems the customers user agent and accept language are not being passed to Komoju.
Only customer IP and e-mail are present. I suspect Spree doesn't pass these options in the gateway request so they returning null values.

https://github.com/komoju/spree_komoju/blob/master/lib/active_merchant/billing/gateways/komoju.rb#L94

**Comment by project-eutopia** [comment]
You are right.  Spree does not seem to store or pass these values onto the gateway.  I am looking in to this to find a way to send these details along to Komoju without adding any columns to the database.
